build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
     implementation("org.apache.httpcomponents:httpmime:4.5.6")
     implementation("gnu.getopt:java-getopt:1.0.13")
     implementation("net.sf.ehcache:ehcache:2.10.6")
+    implementation("ch.qos.logback:logback-classic:1.3.4")
```

Comment:
is there a reason to pin to that specific version?
1.3.5 is the latest in the 1.3 series. 
and 1.4.5 in the latest 1.4. series

Reply:
The decision comes from the work in https://github.com/glencoesoftware/bioformats2raw/pull/167.  The `logback 1.4.x` series requires Java 11 so I kept `logback 1.3.x` for all components which still need Java 8 support
Looks like a new patch release on the 1.3.x line has been released since this work. I can update my PR to use it.

Reply:
That makes sense